client/web/src/enterprise/code-monitoring/components/FormActionArea.tsx

```diff
@@ -163,7 +163,7 @@ export const FormActionArea: React.FunctionComponent<ActionAreaProps> = ({
                 </div>
             )}
             <small className="text-muted">
-                What other actions would you like to do?{' '}
+                What other actions would you like to run?{' '}
```

Comment:
```suggestion
                What other actions would you like to take?{' '}
```

You're absolutely right that "to do" is awkward. I'm not sure "run" is the right choice, though, because it's almost "too far along," if that makes sense. What if it were "to take?"

Reply:
Sounds good to me. A clear net improvement 🙂